src/webview/lib/RecipeWebview.ts

```diff
@@ -92,6 +92,15 @@ class RecipeWebview {
     return this.badgeHandler.safeParseInt(text);
   }
 
+  /**
+   * Find if url contains image
+   *
+   * @param  {string | number | undefined | null} text to be parsed
+   */
+  isImage(url): boolean {
+    return /\.(jpg|jpeg|png|webp|avif|gif|svg)$/.test(url);
```

Comment:
Is this going to return 1/0 or true/false?

Reply:
True/false